### ISSUE TITLE
IP-Adapter-FaceID example training dataset?

### ISSUE THREAD

**Issue by jadechip** [tencent-ailab/IP-Adapter#249]
Since the IP-Adapter-FaceID models are not available for commercial purposes, is there an example dataset for training a IP-Adapter-FaceID model? Would someone be interested in crowd sourcing a model with more a more permissive license?

**Comment by gurteshwar** [comment]
Yes! I would be interested in this.

**Comment by whiterose199187** [comment]
The non-commercial restriction comes from use ofhttps://github.com/deepinsight/insightface. I wonder what are the alternatives available. Would the use of https://github.com/davidsandberg/facenet give comparable results? I am a dev but don't know anything regarding deep learning. 

Anyways, I would be interested in contributing to crowd sourcing a model for commercial use in whatever capacity I can.

**Comment by whiterose199187** [comment]
hello, 

I have a client who is willing to contribute by sponsoring GPU hours for training if anybody is interested.

**Comment by xiaohu2015** [comment]
the model in [facenet](https://github.com/davidsandberg/facenet) maybe also not allowed to commercial purposes

**Comment by whiterose199187** [comment]
hello @xiaohu2015 

I saw the license is MIT and did not see any explicit prohibition of using for commercial purposes, can you please point me to where you read that?

**Comment by thhung** [comment]
@whiterose199187 Yes, I didn't find anything related to the pretrained model in that repo. BTW, how much GPU hours can be offered? We might need to reproduce the dataset first.

**Comment by whiterose199187** [comment]
@thhung do you have an estimate of what would be required? I can accordingly make a proposal to the sponsor with that info

**Comment by thhung** [comment]
> @thhung do you have an estimate of what would be required? I can accordingly make a proposal to the sponsor with that info

@whiterose199187 Sorry for the delayed response. I was on vacation. Is it possible to request this gradually? It depends on several factors (such as GPUs, model size, dataset size, image resolutions, code optimization). It will be clearer once we have the first version end-to-end, which should be small enough to implement quickly. I would aim for a resolution of 512x512. If you would like to discuss further, you can contact me at damienlovedata[at]gmail[dot]com.

**Comment by gabrielhrr** [comment]
@thhung @whiterose199187 any news about the FaceID trained with facenet?
